packages/js/test/plugins/auctionHouseModule/withdrawFromFeeAccount.test.ts

```diff
@@ -35,6 +39,19 @@ test('[auctionHouseModule] withdraw from fee account on an Auction House', async
     feeAccountBalance.basisPoints.toNumber(),
     sol(99).basisPoints.toNumber()
   );
+
+  // And withdrawal destination account got 1 SOL more after withdrawal.
+  const feeWithdrawalDestinationBalance = await mx
+    .rpc()
+    .getBalance(toPublicKey(auctionHouse.feeWithdrawalDestinationAddress));
+
+  t.same(
+    subtractAmounts(
+      feeWithdrawalDestinationBalance,
+      sol(100)
+    ).basisPoints.toNumber(),
+    sol(1).basisPoints.toNumber()
```

Comment:
Can we just check that it's `101`?

Also, let's make it explicit that the `feeWithdrawalDestinationAddress` initially has 100 SOL in the "Given" part of the test.